### DISCUSSION TITLE
Broken Images

### DISCUSSION THREAD

**Discussion by anodeyes** [voidlabs/mosaico#674] [Q&A]
Howdy -

Sorry to be so basic here, but I'm running into a really frustrating issue that I'd appreciate any advice on.

I'm running WordPress 6.11
CIVICRM 5.54.0 
Mosaico 2.10.1665574809

I'm trying to create a basic newsletter. I can drag blocks into the design area and edit text just fine.

However... 

Whenever I try to place an image (whether via upload or gallery) it shows up briefly, and then just reverts to a broken image icon.

I'd be ever so grateful for any suggestions as to how I might go about solving this problem.

Thanks so much.

Joel

**Comment by bago** [comment]
I'm sorry but this doesn't sound like a mosaico bug, but a "civicrm integration" one.
Mosaico 2.10 doesn't even exists here, as you can see in the releases section.

I really don't know anything about the CiviCrm integration and images are handled by backend code, so nothing to do with the mosaico library.

You temporarily see the image because mosaico tries to show you a preview of the image while it is being uploaded: after uploading it will try to get the uploaded image. So, what you see is a failure to upload.

**Comment by anodeyes** [comment]
Thanks for your speedy reply. I appreciate it. 

I understand what you're saying. However, when I look in the Mosaic uploads directory, I see that the images are indeed there (along with the thumbnails as well). 

Any suggestions as what I might do to try to get this working. I'd be very grateful.

Joel

**Comment by bago** [comment]
You can use the browser network inspection tools to see the server reply to the upload and the url requeste by mosaico for the failing image.
